src/overlays/actors/ovl_Obj_Makeoshihiki/z_obj_makeoshihiki.c

```diff
@@ -20,12 +19,71 @@ const ActorInit Obj_Makeoshihiki_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+s32 ObjMakeoshihiki_GetPathIndex(ObjMakeoshihiki* this, GlobalContext* globalCtx) {
```

Comment:
I think this is too close a name to the macro `OBJMAKEOSHIHIKI_GET_PATHLISTINDEX` though they do different things. I feel something along the lines of `ObjMakeoshihiki_GetChildSpawnPointIndex` would be more appropriate.